src/features/bridge/templates/BottomSheetBridgePreview/components/PreviewData/components/PreviewDataItem.tsx

```diff
@@ -0,0 +1,49 @@
+import { NumberUtils } from '@utils/number';
+import { formatUnits } from 'ethers/lib/utils';
+import { View } from 'react-native';
+import { Row, Spacer, Text } from '@components/base';
+import { TokenLogo } from '@components/modular';
+import { COLORS } from '@constants/colors';
+import { scale } from '@utils/scaling';
+import React from 'react';
```

Comment:
try to sort imports in such way:

- React
- react-native
- other npm libs
than import local files